single-audit/single_audit/settings/base.py

```diff
@@ -102,3 +102,8 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
+
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, "static"),
+    '/var/www/static/',
```

Comment:
@fureigh Ooooooh! This was the missing piece for me. I see `/var/www/static` mentioned [here](https://docs.djangoproject.com/en/2.2/howto/static-files/) under "for example", but no explanation of when or why to include it. 

Why is including `/var/www/static/` under `STATICFILES_DIRS` required to make this work in the context of our app?

Reply:
@alexsoble Great question! `os.path.join(BASE_DIR, "static")` is the part that's required in the current context of our app.

`/var/www/static/` is a nod toward a possible production environment, though it may not actually be suitable for cloud.gov... we could scrap line 108 for now and adjust as necessary as part of sorting out deployment.